README.md

```diff
@@ -1,12 +1,30 @@
+
 [![GitHub License](https://img.shields.io/badge/License-Apache%202.0-ff69b4.svg)](https://github.com/Azure/karpenter-provider-azure/blob/main/LICENSE.txt)
 [![CI](https://github.com/Azure/karpenter-provider-azure/actions/workflows/ci.yml/badge.svg?branch=main)](https://github.com/Azure/karpenter-provider-azure/actions/workflows/ci.yml)
 ![GitHub stars](https://img.shields.io/github/stars/Azure/karpenter-provider-azure)
 ![GitHub forks](https://img.shields.io/github/forks/Azure/karpenter-provider-azure)
 [![Go Report Card](https://goreportcard.com/badge/github.com/Azure/karpenter-provider-azure)](https://goreportcard.com/report/github.com/Azure/karpenter-provider-azure)
 [![contributions welcome](https://img.shields.io/badge/contributions-welcome-brightgreen.svg?style=flat)](https://github.com/Azure/karpenter-provider-azure/issues)
 
----
-## Features Overview 
+Table of contents:
+- [Features Overview](#features-overview)
+- [Node Auto Provisioning (NAP) vs. Self-hosted](#node-auto-provisioning-nap-vs-self-hosted)
+- [Known limitations](#known-limitations)
+- [Installation (self-hosted)](#installation-self-hosted)
+  - [Install utilities](#install-utilities)
+  - [Create a cluster](#create-a-cluster)
+  - [Configure Helm chart values](#configure-helm-chart-values)
+  - [Install Karpenter](#install-karpenter)
+  - [Create NodePool](#create-nodepool)
+  - [Scale up deployment](#scale-up-deployment)
+  - [Scale down deployment](#scale-down-deployment)
+  - [Delete Karpenter nodes manually](#delete-karpenter-nodes-manually)
+  - [Delete the cluster](#delete-the-cluster)
+- [Source Attribution](#source-attribution)
+- [Community, discussion, contribution, and support](#community-discussion-contribution-and-support)
```

Comment:
Love the table of contents. We should not sanitize all references to nap, especially since you mention below its the recommended mode. 

Maybe another [Installation (NAP)] section would suffice for this.

Reply:
> We should not sanitize all references to nap, especially since you mention below its the recommended mode.
Agreed, and we don't.

> Maybe another [Installation (NAP)] section would suffice for this.
We reference the page with NAP install instructions instead.